events/event/views.py

```diff
@@ -25,15 +24,20 @@ def get(self, request, event_id=None):
         if not request.user.is_authenticated:
             return PERMISSION_DENIED
         if not event_id:
-            user_id = request.user.id
-            eus = EventUserAssignment.objects.filter(user=user_id)
-            response = [item.event.to_dict() for item in eus]
-            return HttpResponse(json.dumps(response), content_type="application/json")
+            if request.GET.get('q'):
+                query_date = TZ.localize(datetime.utcfromtimestamp(float(request.GET.get('q'))))
```

Comment:
It' s paginator for events

If we receive parameter "q"- we send list with events which start earlier than "q"-parameter